website/docs/getting-started/index.mdx

```diff
@@ -246,27 +242,27 @@ class PostsComponent implements OnInit, OnDestroy {
 
 ```svelte
 <script lang="ts">
-  import { query } from "svelte-apollo";
+  import { query } from 'svelte-apollo'
 
   const postsQueryDocument = gql`
     query Posts {
       posts {
+        id
+        title
+        author {
           id
-          title
-          author {
-            id
-            firstName
-            lastName
-          }
+          firstName
+          lastName
+        }
       }
     }
   `
 
-  const posts = query(postsQueryDocument);
+  const posts = query(postsQueryDocument)
 </script>
 
 <ul>
-    { /* UI */ }
```

Comment:
@jycouet does it valid syntax in svelte? I think comment must be `<!-- UI -->`

Reply:
You are right ` { /* UI */ }` will not work.
If you want a comment in your HTML, you put HTML comment tag like: `<!-- UI -->`

Reply:
While looking at this guide, could we use `KitQL` instead of `Svelte-Apollo` in the example 😇?

Reply:
@dotansimha what do you think?

Reply:
We can do both but we have to keep in mind that guides are here to help most people get started with popular stacks

Reply:
FYI:
https://github.com/timhall/svelte-apollo => It's without code gen
https://github.com/ticruz38/graphql-codegen-svelte-apollo => is the codegen for 👆 svelte-apollo

Adding in the list will probably not fit.